src/katsdpcontroller/generator.py

```diff
@@ -117,6 +117,8 @@ def escape_format(s: str) -> str:
 CONFIG_VOL = scheduler.VolumeRequest("config", "/var/kat/config", "RW")
 #: Number of components in a complex number
 COMPLEX = 2
+#: Autotune fallback behaviour
+KATSDPSIGPROC_TUNE_MATCH = defaults.KATSDPSIGPROC_TUNE_MATCH
```

Comment:
Is it necessary to copy this from `defaults`, as opposed to just referencing `defaults.KATSDPSIGPROC_TUNE_MATCH` when its used? If it was going to be used many times I can see this would save typing, but since it's only used once it's actually more code rather than less.

Reply:
Accepted change.